src/object/class_object.c

```diff
@@ -6765,6 +6774,7 @@ classobj_class_size (SM_CLASS * class_)
 
   size = sizeof (SM_CLASS);
   size += strlen (sm_ch_name ((MOBJ) class_)) + 1;
+  size += strlen (sm_ch_simple_name ((MOBJ) class_)) + 1;
```

Comment:
Since ch_simple_name does not have any real data and only refers to a pointer, isn't it only necessary to increase the size of sizeof(char*)?

`size += sizeof(char*);`

Reply:
Since the memory size is calculated, the size for ch_simple_name can be excluded.
The size of the ch_simple_name pointer is contained in "size = sizeof (SM_CLASS);".

Is it okay to exclude the size of ch_simple_name?

Reply:
I think so.
What is the purpose of the size obtained?

Reply:
It is used in the calculation of "Total bytes for class storage" when printing workspace information in csql.

```
csql> ;info workspace
~
Total bytes for class storage     28668
~
```

Reply:
If it's a simple output of information, it doesn't seem that the calculated value is sensitive.